### ISSUE TITLE
Implement MiunieUserService

### ISSUE THREAD

**Issue by petrspelos** [control-net/Miunie#19]
Miunie user service should provide a `MiunieUser` object instance based on a user's ID.

To persist this data, it uses the `IDataStorage` interface. Because of that, the #18 needs to be finished first.

![image](https://user-images.githubusercontent.com/21304428/50703285-e6155b00-1053-11e9-8de5-2074510fa331.png)

At this point the contents of `MiunieUser` are unknown except for a `DiscordId` property.

We, however, need at least a single property to display. The property should be `Reputation` which needs to be a numeral type of your choosing. Defaults to 0.

:warning: It is also important to create a `MiunieChannel` object that holds `DiscordGuildId` and `DiscordChannelId`.

:information_source: When ideally implementing Unit Tests, please make a new project `Miunie.Storage.xUnit.Tests`. If Unit tests are omitted, please create a new issue for it.
